Gemfile.lock

```diff
@@ -95,9 +95,9 @@ GEM
     grape-entity (0.5.1)
       activesupport
       multi_json (>= 1.3.2)
-    grape-swagger (0.23.0)
+    grape-swagger (0.25.3)
```

Comment:
Hi, thanks for your contribution! Looking over it now. First, do you think you could remove Gemfile.lock from your pull request?